apps/web/app/(app)/(survey-editor)/environments/[environmentId]/surveys/[surveyId]/edit/components/FormStylingSettings.tsx

```diff
@@ -159,14 +109,28 @@ export const FormStylingSettings = ({
 
         <div className="flex flex-col gap-6 p-6 pt-2">
           <div className="flex flex-col gap-2">
-            <ColorSelectorWithLabel
-              label="Brand color"
-              color={brandColor}
-              setColor={setBrandColor}
-              description="Change the brand color of the survey"
+            <FormField
+              control={form.control}
+              name="brandColor.light"
+              render={({ field }) => (
+                <FormItem className="space-y-4">
+                  <div>
+                    <FormLabel className="text-sm font-semibold text-slate-700">Brand color</FormLabel>
```

Comment:
you are adding custom styles the form labels. Since we are working towards a unified design system can we pre-style the FormLabel with the styles we normally use to have a unified look and make the life easier for contributors?